src/pages/instances/InstanceOverviewDeviceList.tsx

```diff
@@ -0,0 +1,63 @@
+import { FC } from "react";
+import { LxdInstance } from "types/instance";
+import { isRootDisk } from "util/instanceValidation";
+import { FormDevice } from "util/formDevices";
+import { LxdDeviceValue, LxdGPUDevice } from "types/device";
+import ResourceLink from "components/ResourceLink";
+
+interface Props {
+  device: LxdDeviceValue;
+  instance: LxdInstance;
+}
+
+const InstanceOverviewDeviceList: FC<Props> = ({ device, instance }) => {
+  return (
+    <div className="u-truncate">
+      {device.type == "disk" ? (
+        isRootDisk(device as FormDevice) ? (
+          <div title={`Pool: ${device.pool}`}>{`pool: ${device.pool}`}</div>
+        ) : (
+          <>
+            <div title={`Storage Volume: ${device.source}`}>
+              storage volume:
+              <ResourceLink
+                type={"volume"}
+                value={device.source as string}
+                to={`/ui/project/${instance.project}/storage/pool/${device.pool}/volumes/custom/${device.source}`}
+              />
+            </div>
+            <div title={`Pool: ${device.pool}`}>
+              pool:
+              <ResourceLink
+                type={"pool"}
+                value={device.pool}
+                to={`/ui/project/${instance.project}/storage/pool/${device.pool}`}
+              />
+            </div>
+          </>
+        )
+      ) : (
+        ""
+      )}
+      <div title={`UID: ${(device as LxdGPUDevice).pci}`}>
```

Comment:
instead of `div`, these should be `span` for better semantic html as it is meant to contain text.